resdk/resources/utils.py

```diff
@@ -94,3 +96,35 @@ def endswith_colon(schema, field):
     """
     if field in schema and not schema[field].endswith(':'):
         schema[field] += ':'
+
+
+def get_resource_id(resource):
+    """Return id attribute of the object if it is resorce, othervise return given value."""
+    return resource.id if isinstance(resource, BaseResource) else resource
+
+
+class resource_list(list):  # pylint: disable=invalid-name
+    """Subclass of ``list`` used for resdk resources.
+
+    This list is aware of resource objects in it and know how to compare
```

Comment:
I would change this "This list is aware of resource objects it holds and knows how to compare them with a list of just object ids."